spec/requests/cats_spec.rb

```diff
@@ -29,4 +29,25 @@
       expect(cat.enjoys).to eq 'Looking at trees'
     end
   end
+
+  describe "meaningful status codes" do
```

Comment:
Would recommend a more descriptive name for your test here.  More specifically than status codes, what is your intent with these tests?